### ISSUE TITLE
Display function.name instead of module.exports

### ISSUE THREAD

**Issue by morkro** [lipp/doclets#165]
I got a couple of helper functions, which are exported as default. Since I have quite a few of them, they only show up as `module/exports()` in the list.

Let's take this example. This is how my code looks like:

```js
**
 * Creates an HTML template string and returns it.
 * @see Code taken from https://hacks.mozilla.org/2015/05/es6-in-depth-template-strings-2/
 * @param {String} template - The HTML string written in template strings
 * @return {String} result - The new string
 */
export default function HTML (template) {
 // ...
}

```
The output in Doclets is like this, among many other `module/exports`:
![screen shot 2016-07-07 at 14 06 38](https://cloud.githubusercontent.com/assets/2461956/16652651/34e713d0-444c-11e6-826b-8ed8a88c705a.png)

This makes it very unorganised and hard to find a function. 
Is it possible to display the `function.name` instead? In my previous example it could be like `module/HTML` or similar.

**Comment by lipp** [comment]
@morkro Yes, this is a bit annoying. I am not sure, if it's ok to behave like this always...
For now, you could just use the @name jsdoc attribute to overwrite JSDoc name deduction.
